connectors/http/rest/src/main/java/io/camunda/connector/http/rest/HttpRequestMapper.java

```diff
@@ -94,9 +94,7 @@ public static HttpRequest toHttpRequest(
       content = new UrlEncodedContent(request.getBody());
     } else {
       content =
-          request.hasBody()
-              ? new JsonHttpContent(GsonComponentSupplier.gsonFactoryInstance(), request.getBody())
-              : null;
+          request.hasBody() ? new JsonHttpContent(new GsonFactory(), request.getBody()) : null;
```

Comment:
I guess we should remove the `GsonFactory`?

Reply:
I don't think we can do it an easy way. The HTTP REST connector uses Google HTTP client, which builds its entity with Gson factory. If we were to decide to get rig of the google http client, it should be handled in a separate task.

WDYT?

Reply:
@igpetrov Looks like it supports jackson: https://googleapis.github.io/google-http-java-client/json.html

Reply:
I see, thanks for pointing it out. I think we should replace this client anyway with a more modern stack. Would be great if you could create a task for that.